src/kernel/lthread/sync/mutex.h

```diff
@@ -27,15 +27,15 @@
  *   to be waken up by mutex->wq when other schedee frees mutex.
  */
 static inline int mutex_trylock_lthread(struct lthread *self,
-					struct mutex *mutex) {
+		struct mutex *mutex) {
 	return WAITQ_WAIT_LTHREAD(self, &mutex->wq, ({
-			int done;
-			done = (mutex_trylock_schedee(&self->schedee, mutex) == 0);
-			if (!done) {
-				mutex_priority_inherit(&self->schedee, mutex);
-			}
-			done;
-		}));
+				int done;
```

Comment:
Wait, it isn't wrong ident at all: we continue code block with ident of two tabs - because it's our desired indent. And then there's indent inside code block, because it's a code block - so we have an indent of one tab. It's also exactly what we want in new blocks of code. So it seems to me that this behaviour is desired.